packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1031,7 +1037,25 @@ export function diffHydratedProperties(
             warnForPropDifference(propKey, serverValue, expectedStyle);
           }
         }
-      } else if (isCustomComponentTag) {
+      } else if (
+        enableCustomElementPropertySupport &&
+        isCustomComponentTag &&
+        (propKey === 'offsetParent' ||
+          propKey === 'offsetTop' ||
+          propKey === 'offsetLeft' ||
+          propKey === 'offsetWidth' ||
+          propKey === 'offsetHeight' ||
+          propKey === 'isContentEditable')
+      ) {
+        // $FlowFixMe - Should be inferred as not undefined.
+        extraAttributeNames.delete(propKey.toLowerCase());
+        if (__DEV__) {
```

Comment:
This checks seems unnecessary because of the outer one. Or does lint complain?

Reply:
Yeah, I get this error from `yarn linc` if I remove the `__DEV__` check:
```
/usr/local/google/home/jarhar/react/packages/react-dom/src/client/ReactDOMComponent.js
  1052:9  error  Wrap console.error() in an "if (__DEV__) {}" check  react-internal/no-production-logging

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```